cEP-0020.md

```diff
@@ -0,0 +1,501 @@
+- Number of answers given in `coala/coala` room
+- Number of questions asked in `coala-bears` room
+- Number of  answers given in `coala-bears` room
+- Number of questions asked in `coala/corobo` room
+- Number of answers given in `coala/corobo` room
+- Number of questions asked in `coala/community` room
+- Number of answers given in `coala/community` room
+- Number of messages in coala other rooms
+- Make use of `corobo` on Gitter
+- Introduce `.coafile` in user own repository
+- Introduce `.coafile` in an organisation with 50 or more stars
+  for its repositories
+- Introduce CI in the users own repository to run coala using Travis CI
+- Setup the CIs used by coala to your fork
+
+### Points
+
+Points are a simple tool to reward reward positive contributions to coala,
+and they provide an excellent base upon which to build a gamified design.
+Now let's assign some points to each of the tracked activities mentioned
+above.
+
+But, as we are assigning points to the issues based on their labels which are
+initially labeled by the Gitmate-bot, we can not be sure all the
+time if the issue
+was labeled correctly or not. So, an issue should have three `Thumbs Up`
+reactions to qualify to be tracked in this system.
+
+Mainly it will be the developers' job to verify every new
+opening issues and give it
+the `Thumbs Up`, but in case if an issue seems to be
+labeled inappropriately by the bot,
+developers should request a maintainer to label it correctly and
+only then give `Thumbs Up` to the issue.
+
+Similarly in case of the pull requests if the mentioned issue
+in the commit is opened
+by newcomer and doesn't have three `Thumbs Up`, it will not
+qualify to be tracked in this system.
+
+- Create Issue
+  - 20 points for type/bug
+  - 20 points for bear proposal
+  - 10 points for difficulty newcomer
+  - 20 points for difficulty low
+  - 30 points for difficulty medium
+- Create Pull Requests
+  - All the points will be calculated and assigned when the pull
+    request will be merged.
+  - If there are no Gitmate errors
+    - 20 points for bug fixes
+    - 20 points for writing a bear
+    - 10 points for difficulty newcomer
+    - 20 points for difficulty low
+    - 30 points for difficulty medium
+  - If Gitmate errors are b/w 1 to 5
+    - 15 points for bug fixes
+    - 15 points for writing a bear
+    - 7 points for difficulty newcomer
+    - 15 points for difficulty low
+    - 20 points for difficulty medium
+  - If Gitmate errors are b/w 5 to 10
+    - 10 points for bug fixes
+    - 10 points for writing a bear
+    - 5 points for difficulty newcomer
+    - 10 points for difficulty low
+    - 15 points for difficulty medium
+  - If Gitmate errors are b/w 10 and beyond
+    - No points given
+- Activities related to introducing `.coafile`
+  - 10 points
+- Activities related to setting up CIs to the fork
+  - 10 points
+
+Note that in case of an issue labeled as bug and also has a difficluty level,
```

Comment:
*labeled* -> *labelled* (in British English - the one we use here)
Also, **a** bug.
*difficluty* -> *difficulty*